src/reactor.js

```diff
@@ -92,6 +92,48 @@ class Reactor {
     return this.__changeObserver.onChange(getter, handler)
   }
 
+  /**
+   * Removes a change observer that has previously been set
+   *
+   * 1. unobserve(keyPath) Removes all observables for keyPath
+   * 2. unobserve(getter) Removes all observables for getter and getter dependencies
+   * 3. unobserve(keyPath|getter, handlerFn) Removes the specific handlerFn for
+   *    the passed in keyPath or getter.
+   *
+   * @param {KeyPath|Getter} getter
+   * @param {function} handler
+   */
+  unobserve(getter, handler) {
+    var keyPath
+
+    if (KeyPath.isKeyPath(getter)) {
+      keyPath = getter
+    }
+
+    // Returns unwatchFn if no handler, or if handler is the observer's handler
+    // Otherwise returns `null`
+    var unwatchIfShould = entry => {
+      if (handler) {
+        if (handler === entry.handler) {
+          return entry.unwatchFn
+        }
+
+        return null
+      }
+
+      return entry.unwatchFn
+    }
+
+    each(this.__changeObserver.__observers.map(entry => {
+      if (keyPath && KeyPath.eql(keyPath, Getter.getDeps(entry.getter)[0])) {
```

Comment:
Kind of confused what this line is doing.

If the getter is a keyPath then we are comparing against the first dep of the `enter.getter`?

It seems like this should be something like:

```js
const isKeyPath = keyPath.isKeyPath()

// unwatchifShould invokes entry.unwatchFn if found

each(this.__changeObserver.__observers.map(entry => {
  if (isKeyPath && KeyPath.isKeyPath(entry.getter) && KeyPath.eql(getter, entry.getter)) {
    return unwatchIfShould(entry)
  } 
  if (entry.getter === getter) {
    return unwatchIfShould(entry)
  }
}), unwatchFn => unwatchFn && unwatchFn())
```

Also are you using `map` and `each` as to not mutate the `__observers` map while iterating?

Lastly I think this logic should be in the `ChangeObserver` under `__changeObserver.unobserve`.  IMO the reactor shouldn't have knowledge of the inner workings of `ChangeObserver`